src/pheval/analyse/analysis.py

```diff
@@ -560,9 +768,17 @@ def benchmark_runs(
     show_default=True,
     help="Specify analysis for variant prioritisation",
 )
+@click.option(
+    "--disease-analysis/--no-disease-analysis",
+    default=False,
+    required=False,
+    type=bool,
+    show_default=True,
+    help="Specify analysis for disease prioritisation",
+)
 @click.option(
     "--plot-type",
-    "-p",
+    "-pt",
```

Comment:
By convention, I think it is discouraged to use more than one letter for the short form, as standard CLI tools will read -pt as `-p -t` (short option stacking). I would recommend an unrelated character instead, like `t` or `y`.